src/app/hooks/useTransformToOasisAddress.ts

```diff
@@ -8,6 +8,10 @@ export const useTransformToOasisAddress = (ethOrOasisAddress: string): string |
     let shouldUpdate = true
 
     const transformToOasisAddress = async (addr: string) => {
+      if (!ethOrOasisAddress) {
```

Comment:
where is empty string coming from?

Reply:
Search, this hook I believe `useRuntimeAccountConditionally`

Reply:
Ah, otherwise we would need to disable transformation until `options.query.enabled`